scripts/rest_examples/ib_list.http

```diff
@@ -0,0 +1,5 @@
+// @no-log
+GET https://console.stage.redhat.com/api/image-builder/v1/composes HTTP/1.1
+Content-Type: application/json
+Authorization: Basic user:password
+
```

Comment:
FYI this will only work with the HTTP proxy. Unfortunately, HTTP proxy cannot be configured via this format. Maybe drop a comment explaining that.

Reply:
Yes, I configured my proxy on my IDE, we should add it to the request, I'll add it on my next push.

Reply:
you can configure a proxy in vscode easily, in settings -> `http.proxy` 
extensions (like http clients) automatically use it

Reply:
Yeah I know I was just crying that the .http format does not have support for that. I wish.